Bổ sung format câu hỏi

phamquocbuu opened the pull request:
Mới tìm thấy cái link Markdown syntax của Github, ngay dưới ô comment :v: 
https://guides.github.com/features/mastering-markdown/

phamquocbuu committed:
Bổ sung format câu hỏi

Tiêu đề và nội dung

phamquocbuu committed:
Bổ sung link markdown syntax

chungth commented on README.md:
```diff
@@ -21,4 +21,36 @@ Tất nhiên để thay đổi thói quen không phải là chuyện dễ dàng,
+- `[Package] Xử lý Header như thế nào?`
+
+### Nội dung
+Phải theo các quy ước sau để người khác dễ hỗ trợ:
```
@cosmospham  Từ `phải` nghe có vẻ hơi nặng nề , e chuyển qua dùng từ `cần` có được ko ? 

> Cần theo các quy ước dưới đây để thành viên khác có thể dễ dàng hơn trong việc hỗ trợ bạn.

chungth commented on README.md:
````diff
@@ -21,4 +21,36 @@ Tất nhiên để thay đổi thói quen không phải là chuyện dễ dàng,
+?>
+```
+- Code indent (thụt dòng) vào bằng 4 khoảng trắng (4 space)
+- Cái gì copy được thì nên copy vào nội dung câu hỏi - thay vì chụp màn hình. Vì text thì search dễ hơn (rất nhiều) so với hình. Google với Github không index được hình đâu.
````
@cosmospham  đoạn này a nghĩ là nên chỉnh lại một chút 

> Cái gì copy được thì nên copy vào nội dung câu hỏi - thay vì chụp màn hình. Vì text thì search dễ hơn (rất nhiều) so với hình. Google với Github không index được hình đâu.

=> 

> Trong trường hợp bạn gặp lỗi cần debug, song song với việc chụp màn hình, hãy copy cả những đoạn lỗi vào nội dung câu hỏi để những bạn khác nếu gặp lỗi này có thể dùng Google search hay Gihub search để tìm ra, tránh việc đặt câu hỏi thừa không cần thiết.

phamquocbuu committed:
Fix - Format câu hỏi

Fix theo gợi ý của @chungth; sửa vài chỗ cho đầy đủ hơn; đổi link Markdown syntax bằng link của Github
